.gitignore

```diff
@@ -7,3 +7,4 @@
 .pytest_cache
 __pypackages__
 .python-version
+.isort.cfg
```

Comment:
Seems like this is one of those config files that you would want to share with all developers, so everyone gets a consistent isort experience. (Plus I want to see it, lol)